<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualEndoscopy</class>
 <widget class="qMRMLWidget" name="VirtualEndoscopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>286</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>The Virtual Endoscopy Pipeline Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputOriginalImageSelector1">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seeds along the axis:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="treeFiducialsNodeSelector">
        <property name="toolTip">
         <string>Select a fiducial list to define control points for the axis.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
           <string>vtkMRMLMarkupsFiducialNode</string>
           <string>vtkMRMLAnnotationHierarchyNode</string>
           <string>vtkMRMLFiducialListNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lumen Label Volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="outputLumenLabelImageSelector1">
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
           <!-- <string>vtkMRMLScalarVolumeNode</string> -->
           <string>vtkMRMLLabelMapVolumeNode</string>
         </stringlist>
        </property>
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Calcificatoin:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSliderWidget" name="calcificationSlicerWidget">
        <property name="toolTip">
         <string>Set threshold value for calcification. Voxels that's brighter than this value will be considered as calcification.</string>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>800.0000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Threshold:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkSliderWidget" name="thresholdSliderWidget">
        <property name="toolTip">
         <string>Threshold Value.</string>
        </property>
        <property name="singleStep">
         <double>1</double>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Vessel brighter than the surroundings?</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="vesselBrighterCheckBox">
        <property name="toolTip">
         <string>Is vessel brighter than the surroundings?</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Perform super-resolution segmentation?</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
        <widget class="QCheckBox" name="superResolutionCheckBox">
          <property name="toolTip">
            <string>Perform super-resolution segmentation?</string>
          </property>
          <property name="checked">
            <bool>false</bool>
          </property>
        </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
     <widget class="QPushButton" name="computeWholeProcess1">
       <property name="toolTip">
         <string>Run the whole process.</string>
       </property>
       <property name="text">
         <string>Run It!!!!!!</string>
       </property>
     </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <connections>
  <connection>
   <sender>VirtualEndoscopy</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputOriginalImageSelector1</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VirtualEndoscopy</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>treeFiducialsNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VirtualEndoscopy</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputLumenLabelImageSelector1</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
